app/src/main/java/swati4star/createpdf/fragment/FAQFragment.java

```diff
@@ -42,6 +47,41 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         View view = inflater.inflate(R.layout.fragment_faq, container, false);
 
+        mContext = view.getContext();
+
+        TextView categoryView = view.findViewById(R.id.faq_category_header);
+        categoryView.setText(mCategory);
+
+        String[] categories = mContext.getResources().getStringArray(R.array.faq_categories);
+        TabLayout tabLayout = (TabLayout) view.findViewById(R.id.faq_category_tab_layout);
+
+        // For FAQ items without a category
+        tabLayout.addTab(tabLayout.newTab().setText("All"));
+
+        for (String category : categories) tabLayout.addTab(tabLayout.newTab().setText(category));
+
+
+        tabLayout.setOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
```

Comment:
This is deprecated. With more time I would implement an updated version and clean up some of this code